index.html

```diff
@@ -1,12 +1,54 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,400;0,500;1,400;1,500&display=swap"
+      rel="stylesheet"
+    />
     <title>Document</title>
-</head>
-<body>
-    
-</body>
-</html>
\ No newline at end of file
+  </head>
+
+  <body>
+    <header>
+      <nav class="header">
+        <img src="./imgs/logo.png" alt="logo" />
+        <div class="burger-menu">
+          <span class="line"></span>
+          <span class="line"></span>
+          <span class="line"> </span>
+        </div>
+      </nav>
+    </header>
+    <main>
+      <section class="hero">
+        <div class="image"></div>
+        <div class="group">
+          <h1>Tonic</h1>
+          <div class="tags-holder">
+            <h4 class="canopy">canopy</h4>
+            <div class="dot"></div>
+            <span class="tag">Back End Dev</span>
+            <div class="dot"></div>
+            <span class="tag">2022</span>
+          </div>
+        </div>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <div class="skills">
+          <button class="btn-skill">html</button>
+          <button class="btn-skill">css</button>
+          <button class="btn-skill">javaScript</button>
+        </div>
+        <button class="view-project">See project</button>
+      </section>
```

Comment:
### Your Section
![Screenshot from 2022-09-21 17-49-44](https://user-images.githubusercontent.com/93039359/191509424-c8e5755b-d217-4962-937e-f3298afb9862.png)



- [ ] You have added a wrong section. The section which you have added is for the next project. But for this project you need to add the section which is shown below :arrow_down:

![expected](https://raw.githubusercontent.com/microverseinc/curriculum-html-css/main/portfolio/images/m1_setup/headline-mobile.png?token=GHSAT0AAAAAABYX6HCNEVNK4A4PXFAVHOUMYZLCWIA)

Reply:
@HAMMAS-SALEEM  I have the option to choose my template it's mentioned in the requirements!